eslint.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = [
     plugins: {
       eslintPlugin,
       importPlugin,
-      filenames,
+      filenamesPlugin: fixupPluginRules(filenames),
```

Comment:
does it matter that it's not a string like `'i18n-text'` on the next line?

Reply:
That should be okay - what I ended up doing was bringing in the filenames match regex rule into github. The filenames plugin is no longer supported and the compat utility was not working for our rule configuration.